diff --git a/node_modules/@types/ramda/index.d.ts b/node_modules/@types/ramda/index.d.ts
index 2f3beec..bb94c9b 100644
--- a/node_modules/@types/ramda/index.d.ts
+++ b/node_modules/@types/ramda/index.d.ts
@@ -743,6 +743,10 @@ export function inc(n: number): number;
  * Given a string, this function checks for the string in another string or list and returns
  * a boolean.
  */
+export function includes(__: Placeholder, list: string): (a: string) => boolean;
+export function includes<T>(__: Placeholder, list: readonly T[]): (a: T) => boolean;
+export function includes(__: Placeholder): (list: string, a: string) => boolean;
+export function includes<T>(__: Placeholder): (list: readonly T[], a: T) => boolean;
 export function includes(s: string, list: readonly string[] | string): boolean;
 export function includes(s: string): (list: readonly string[] | string)  => boolean;
 export function includes<T>(target: T, list: readonly T[]): boolean;
